packages/gatekeeper/dist/ducks/gatekeeper.js

```diff
@@ -8,7 +8,8 @@ Object.defineProperty(exports, "__esModule", {
 exports["default"] = reducer;
 exports.getResult = getResult;
 exports.getSuccess = getSuccess;
-exports.recordResult = exports.RECORD = exports.initialState = exports.reducerName = void 0;
+exports.isWorking = isWorking;
```

Comment:
Could we please document the intended usage of `isWorking`?

Reply:
done